src/content/patterns/creation-flows/creation-flows.mdx

```diff
@@ -0,0 +1,164 @@
+- [Multistep creation](#multistep-creation)
+- [confirmation](#confirmation)
+
+</AnchorLinks>
+
+
+## Standard Creation
+
+<AnchorLinks small>
+
+- [Modal](#modal)
+- [Side panel](#side-panel)
+- [Full screen](#full-screen)
+- [Advanced option](#advanced-option)
+
+</AnchorLinks>
+
+### Modal
+If there are only 1 or 2 options involved in the creation then use a small, standard modal, overlaying the content where the creation was initiated. Scrolling inside a small modal should be avoided. 
+
+<ImageComponent cols="8">
+
+![Example](images/2.png)
+
+</ImageComponent>
+
+
+### Side panel
+Consider using a side panel to take advantage of the added real-estate and keep the user in context of where they were working. You may choose to use either the slide in or slide over panels.
+
+
+<ImageComponent cols="8">
+
+![Example](images/3.png)
+
+</ImageComponent>
+
+### Full page
+Consider using this when wanting to take full advantage of the screen’s available real estate. 
+
+<ImageComponent cols="8">
+
+![Example](images/4.png)
+
+</ImageComponent>
+
+<AnchorLinks small>
+
+- [Button position](#button-position)
+- [Image](#Image)
+- [Advanced option](#advanced-option)
+
+</AnchorLinks>
+
+#### Button position
+When the full page dialog is viewed in mobile, the buttons are positioned at the bottom of the screen.
+
+<ImageComponent cols="8">
+
+![Example](images/5.png)
+
+</ImageComponent>
+
+On a larger screen it is recommended that buttons always remain visible to the user.
+
+When there are enough fields going beyond the fold, buttons should be located in the top right and should stick upon scrolling. If only a couple are present, buttons can be placed on the left below the last field. 
+
+<ImageComponent cols="8">
+
+![Example](images/6.png)
+
+</ImageComponent>
+
+#### Image
+To include a supporting image, place it on the right hand side of the window. 
+
+<ImageComponent cols="8">
+
+![Example](images/7.png)
+
+</ImageComponent>
+
+#### Advanced Option
+
+The advanced option is used to avoid overwhelming users, whilst allowing advanced users to access everything. It can be applied to the side panel and the full page dialog.
+
+Consider by default having a minimal set of options visible, with the ability for the user to chose if they want to expose more ‘advanced’ options.
+
+If the user switches to advanced mode, ideally remember this for when they perform the same task.
+
+<ImageComponent cols="4">
+
+![Example](images/8.png)
+
+</ImageComponent>
+<br />
+<ImageComponent cols="8">
+
+![Example](images/9.png)
+
+</ImageComponent>
+
+
+## Multistep creation
+If the creation task is complex, consider whether breaking the process up into steps could help the user.  Consider displaying related or dependant options together.
+
+Multistep creation follows the same principles as Standard creation, with the addition of multisteps.
+
+
+<AnchorLinks small>
+
+- [Side panel](#side-panel)
+- [Full page](#full-page)
+
+</AnchorLinks>
+
+### Side panel
```

Comment:
Here I think you can maybe repeat the benefits of having it in a side panel like in the "standard creation section" incase the user doesn't read that.